src/scripts/main.js

```diff
@@ -1 +1,20 @@
 'use strict';
+
+const mainImg = document.querySelector('.gallery__large-img');
+const galleryList = document.querySelector('.gallery__list');
+
+galleryList.addEventListener('click', (myEvent) => {
+  myEvent.preventDefault();
+
+  const img = myEvent.target.closest('.list-item__link');
+
+  // but working without this, do I need to add it?
+  if (!galleryList.contains(img)) {
```

Comment:
Yep, it works here, but it's always better to cover all possible cases and prevent unexpected behavior/errors

I'll approve this PR, but in the future don't ask smth via code comments, better add your own comment with the question on GitHub